docs/reference/http-api.md

```diff
@@ -66,37 +66,40 @@ Load the state of a document given its DocID.
     }
     ```
 
-### Create document
-Create a new document, or load a document from its genesis content. The genesis content may be signed (a DagJWS for the `tile` doctype), or unsigned in some cases.
+### Create stream
+
+**:octicons-alert-16: Disabled in gateway mode**
+
+Create a new stream, or load a stream from its genesis content. The genesis content may be signed (a DagJWS for streams of type TileDocument), or unsigned in some cases.
 
 === "Request"
-    
+
     ```bash
-    POST /api/v0/documents
+    POST /api/v0/streams
     ```
 
     #### Request body fields:
 
-    - `doctype` - the name of the doctype to use (e.g. 'tile'), string
-    - `genesis` - the genesis content of the document (will differ per doctype)
-    - `docOpts` - options for the document creation, [DocOpts](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_common.docopts-1.html){:target="_blank"} (optional)
+    - `type` - the type code of the streamtype to use (e.g. `0` for TileDocuments). Type codes for the supported stream types can be found [in this table](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-59/tables/streamtypes.csv).
```

Comment:
I think this also works if you pass a string. (not necessary to document though).